src/m365/spo/commands/site/site-set.ts

```diff
@@ -356,60 +353,53 @@ class SpoSiteSetCommand extends SpoCommand {
             verbose: this.verbose
           });
         }, operation.PollingInterval);
-      }
-    });
+      });
+    }
   }
 
-  private updateSiteOwners(logger: Logger, args: CommandArgs): Promise<void> {
+  private async updateSiteOwners(logger: Logger, args: CommandArgs): Promise<void> {
     if (!args.options.owners) {
-      return Promise.resolve();
+      return;
     }
 
-    return new Promise<void>((resolve: () => void, reject: (error: any) => void): void => {
-      if (this.verbose) {
-        logger.logToStderr(`Updating site owners ${args.options.url}...`);
-      }
+    if (this.verbose) {
+      logger.logToStderr(`Updating site owners ${args.options.url}...`);
+    }
 
-      Promise.all(args.options.owners!.split(',').map(o => {
-        return this.setAdmin(args.options.url, o.trim());
-      }))
-        .then((): void => {
-          resolve();
-        }, (err: any): void => {
-          reject(err);
-        });
-    });
+    try {
```

Comment:
Let's remove this try catch block, as it is throwing anyway. It will end up in the parent try catch